doc/data/messages/i/implicit-flag-alias/good.py

```diff
@@ -0,0 +1,7 @@
+from enum import IntFlag
+
+
+class FilePermissions(IntFlag):
+    READ = 1
+    WRITE = 2
+    EXECUTE = 4
```

Comment:
Kind of a boring example.  More interesting suggestions welcome.

Reply:
It's nice and grounded in reality which help to understand 👍